src/state/stake/hooks.ts

```diff
@@ -721,52 +721,16 @@ export function useTotalPngEarned(): TokenAmount | undefined {
   const chainId = useChainId()
 
   const png = PNG[chainId]
-  const stakingInfo0 = useStakingInfo(0)
-  const stakingInfo1 = useStakingInfo(1)
-  const stakingInfo2 = useMinichefStakingInfos(2)
```

Comment:
what was the reason to remove this? may be we can remove stakingInfo0 and stakingInfo1 but stakingInfo2 is important to calculate total png

Reply:
@Shungy we still want to keep `const stakingInfo2 = useMinichefStakingInfos(2)` because now it will not hurt the page performance as we are just calling it in modal